Makefile

```diff
@@ -111,7 +111,7 @@ v8:
 	tools/make-v8.sh v8
 	$(MAKE) -C deps/v8 $(V8_ARCH).$(BUILDTYPE_LOWER) $(V8_BUILD_OPTIONS)
 
-test: | cctest  # Depends on 'all'.
+test: | build-addons cctest  # Depends on 'all'.
```

Comment:
The `test` target doesn't actually test `addons`, so the addons don't need to be built?